_includes/v22.2/cdc/csv-changefeed-format.md

```diff
@@ -0,0 +1 @@
+{% include_cached new-in.html version="v22.2" %} Changefeeds emit the same CSV format as [`EXPORT`](export.html). In v22.1, changefeeds emitted CSV data that wrapped some values in single quotes, which were not wrapped when exporting data with the `EXPORT` statement.
```

Comment:
nit: we can just say `Previously, changefeeds emitted ...` to avoid worrying about whether previous behavior also existed in v21.2 or earlier. Pinning to a specific previous release suggests that it only applies to that singular previous release.

Reply:
The reason I pinned this was because I knew that the `format='csv'` option only became available in v22.1. You feel it is still necessary to change in that case?

Reply:
Gotcha, not at all then! Thanks for clarification.